docs/src/pages/company/software-engineer/software-engineer.md

```diff
@@ -0,0 +1,87 @@
+- **A track record demonstrating an eye for design and solving real world user problems**. If you have a knack for shipping beautiful, intuitive software, we want you on our team.
+- **Experience building and shipping production code in a team setting** with a passion for writing tested, performant, and high-quality code.
+- **Strong written and verbal communication skills**. As part of our open source team, you’d interface both directly and indirectly with community members, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+- **Ability to dive into complex problems**. You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
+- **Ready and willing to ask and answer questions**. If you’re comfortable saying you’re unsure, asking for help, and reaching out to assist others, you’ll be an incredible addition to our team. Our team thrives because of continuous learning. First time mistakes should be celebrated, not blamed.
+- **Avoid monolithic deliverables**. You scope and stage your work into well-defined milestones to ship.
+- **Past work with frontend infrastructure**. Perhaps you’ve created your company’s design system, written a babel plugin, or written complex React components. It would be great if you could address this in your cover letter!
+
+## Experience it would be nice if you had, but isn’t required
+
+- **You’ve maintained an active repository before**. Maybe you’ve helped maintain a popular open source repo, or maybe you’ve worked on internal repos that saw contributions from multiple teams. Previous experience with highly active repo workflows is a definite plus for this role.
+- **You have used Material-UI before**. You have built a non trivial application with Material-UI in the past. You know the limitations of the library, you know a few areas that could be improved.
+- **You have contributed code to Material-UI before**. A history of contributing to Material-UI would be a definite plus.
+
+## The best parts of this job
+
+- **You’ll be at the cutting edge of application development** — working on one of the fastest-growing UI frameworks on the market.
+- **You’ll be part of an active, open, friendly community** of developers that are really excited about building awesome applications.
+- **Your role will be key to making Material-UI the go to UI framework solution** for building applications and design systems with React.
+
+## The worst parts of this job
+
+- **Shifting context.** You may necessarily have to shift context and dive into a different feature before the current one is done. It may even be in an area of the code base you’re unfamiliar with or don’t have a ton of understanding about. It’s fun, rewarding work, but it can be very challenging.
```

Comment:
```suggestion
- **Shifting context.** You will necessarily have to shift context and dive into a different feature before the current one is done. It may even be in an area of the code base you’re unfamiliar with or don’t have a ton of understanding about. It’s fun, rewarding work, but it can be very challenging.
```